pages/App/Home.js

```diff
@@ -0,0 +1,87 @@
+import Image from 'next/image'
```

Comment:
Created components outside of pages directory. Because every under pages is available as a route.

Reply:
resolved